src/Sdk/DTWebApi/WebApi/TimelineRecord.cs

```diff
@@ -209,35 +217,46 @@ public TimelineReference Details
             set;
         }
 
-        [DataMember(Order = 40)]
-        public Int32? ErrorCount
+        public Int32 ErrorCount
         {
-            get;
-            set;
+            get
+            {
+                return m_errorCount.GetValueOrDefault(0);
+            }
+            set
+            {
+                m_errorCount = value;
+            }
         }
 
-        [DataMember(Order = 50)]
-        public Int32? WarningCount
+        public Int32 WarningCount
         {
-            get;
-            set;
+            get
+            {
+                return m_warningCount.GetValueOrDefault(0);
+            }
+            set
+            {
+                m_warningCount = value;
+            }
         }
 
-        [DataMember(Order = 55)]
-        public Int32? NoticeCount
+        public Int32 NoticeCount
         {
-            get;
-            set;
+            get
+            {
+                return m_noticeCount.GetValueOrDefault(0);
+            }
+            set
+            {
+                m_noticeCount = value;
+            }
         }
 
         public List<Issue> Issues
         {
             get
             {
-                if (m_issues == null)
-                {
-                    m_issues = new List<Issue>();
-                }
```

Comment:
`m_issues`, `m_previousAttempts`, and `m_variables` are now instantiated once and only once within `EnsureInitialized`.

I wanted to declare them as `readonly` but had to stop short of that because of the quirks of `DataContractSerializer`